deisctl/client/client.go

```diff
@@ -74,13 +74,19 @@ func (c *Client) UpgradePrep(argv []string) error {
 	usage := `Prepare platform for graceful upgrade.
 
 Usage:
-  deisctl upgrade-prep [options]
+  deisctl upgrade-prep [--stateless]
```

Comment:
Yeah, auto-detect could be possible down the road. I personally like the explicit flag for now.